src/lighteval/tasks/registry.py

```diff
@@ -239,15 +279,17 @@ def taskinfo_selector(
         if suite_name not in DEFAULT_SUITES:
             hlog(f"Suite {suite_name} unknown. This is not normal, unless you are testing adding new evaluations.")
 
-        # Store few_shot info for each task name (suite|task)
-        few_shot_dict[f"{suite_name}|{task_name}"].append((few_shot, truncate_few_shots))
+        # This adds support for task supersets (eg: mmlu -> all the mmlu tasks)
+        for expanded_task in task_registry.expand_task_definition(f"{suite_name}|{task_name}"):
+            # Store few_shot info for each task name (suite|task)
+            few_shot_dict[expanded_task].append((few_shot, truncate_few_shots))
```

Comment:
What happens if we pass both `custom|mmlu|3,custom|mmlu:abstract_algebra|5` for example?